source/user-manual/user-administration/password-management.rst

```diff
@@ -179,7 +198,7 @@ The options ``-au`` and ``-ap`` are necessary to change the passwords for the AP
 Changing the passwords in a distributed environment
 ---------------------------------------------------
 
-Follow the instructions below to change the passwords for all the Wazuh indexer users as well as the Wazuh manager API users.
+Follow the instructions below to change the passwords for all Wazuh indexer users, Wazuh manager API users, and the Wazuh dashboard user.
```

Comment:
The "Wazuh dashboard user" isn't evident from reading the introduction. Does it mean `wazuh-wui`? `admin`? `kibanaserver`? Other?